conf/configuration.go

```diff
@@ -17,6 +17,7 @@ type OAuthProviderConfiguration struct {
 	Secret      string `json:"secret"`
 	RedirectURI string `json:"redirect_uri" split_words:"true"`
 	URL         string `json:"url"`
+	ApiURL      string `json:"api_url"`
```

Comment:
added a new field to include a custom api endpoint for oauth providers in the event future providers are similar to azure (where the auth and api endpoints are different)